libraries/lib-wave-track/WaveClip.cpp

```diff
@@ -136,6 +136,13 @@ WaveClip::~WaveClip()
 {
 }
 
+AudioSegmentSampleView
+WaveClip::GetSampleView(size_t ii, sampleCount start, size_t length) const
+{
+   assert(ii < GetWidth());
```

Comment:
I do not like new assertions with no accompanying comments to explain why they are justified.  Assertions should document assumptions you are making and explain them to the user.

I consider an assumption at the top of a function to be justified if the header file has a `@pre` Doxygen comment.